CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Validate the cache to prevent pulling stale cache objects ([giantswarm/giantswarm/9747](https://github.com/giantswarm/giantswarm/issues/9747))
+### Added
+
+- Validate the cache to prevent pulling stale cache objects.
```

Comment:
@yasn77 Thanks again for adding this. I'm going to cut a 3.1.0 release once this is merged.

I just removed the link as normally we don't include since the issue is private.